README.md

```diff
@@ -15,24 +14,21 @@ no widely-used mobile application that would allow that person to
 store lyrics (text) and melodies (audio) in a single, organized place.
 **dtty** was created to solve this problem.
 
-___________________________________
-
+---
 
 ## Technologies Used
 
-    - ReactJS
-    - MediaStream Recording API
-    - Firebase Cloud Storage
-    - JSON Server
-    - MomentJS
-    - Semantic UI
-    - Fort Awesome
-
-
-_____________________________________
+- React
```

Comment:
Markdown is a beautiful markup language.... some little quirks like this extra indention you had made it look a little weird, like it wanted to look like code instead of bullet points. By removing the indentation, the list now looks a little nicer. 😎